### PR TITLE
Update phpunit_bridge.rst

### PR DESCRIPTION
**Pull request by ro0NL**
see https://github.com/symfony/symfony/pull/34566

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by ro0NL** [comment]
Note the PR is not merged, and perhaps will not be also: https://github.com/symfony/symfony/pull/34566#issuecomment-557875366

**Comment by wouterj** [comment]
I believe Nicolas is reacting on your "Can we please use PHPUnit 9 in 5.1?" question.

Anyway, the yellow "Waiting Code Merge" label indicates that we'll not merge this PR before the upstream PR is merged.

**Pull request closed without merging**
